client.go

```diff
@@ -4,49 +4,35 @@ import (
 	"bytes"
 	"encoding/xml"
 	"errors"
+	"fmt"
 	"io"
 	"io/ioutil"
-	"log"
 	"mime"
 	"mime/multipart"
-	"net"
 	"net/http"
 	"reflect"
 	"strings"
-	"time"
 )
 
-// ClientDialTimeout default timeout 30s
-var ClientDialTimeout = time.Duration(30 * time.Second)
-
 // UserAgent is the default user agent
-var UserAgent = "go-soap-0.1"
+const UserAgent = "go-soap-0.1"
```

Comment:
the var here was actually intentional and I think it makes sense to allow users of the client to set their own user agent

Reply:
Fixed